build_config/lib/src/pubspec.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Should we consider publishing this as a separate package?

Reply:
I have seen several community members ask for a package like this, but at the same time I don't know that we really want to support it.

Reply:
I think I'd be happy to switch this package over to use something we publish later - I don't think I want to try to use this as the starting point for a new package though.

Reply:
sgtm